xero.gg | Added "Challenges" page.

Dekirai opened the pull request:
Whats new: The website has added a "Challenges" page so I updated my presence to make it work with the new website update.
Proof of it working: [Video](https://streamable.com/47nr6e)

ririxidev commented on websites/X/Xero.gg/presence.ts:
```diff
@@ -96,6 +96,17 @@ presence.on("UpdateData", async () => {
+  } else if (document.location.pathname.includes("/challenge")) {
+    presenceData.details = "Viewing a page:";
+    presenceData.state = "Challenges";
+    try {
+      const challenges_title = document.querySelector(
+        "#settings-data-container > div.xero-header-standalone"
+      ).textContent;
+      presenceData.state += ` (${challenges_title})`;
+    } catch {
+      //Catch nothing
+    }
```
If you use try function, why you don't catch anything? Just use something like this:
```javascript
const f = hh() ? jj() : ""
```

Dekirai commented:
I just removed the trycatch since it was useless at this point anyway. Don't even know why I've added it.
Also made small changes to other trycatches where I catched nothing.
